app/controllers/position_histories_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PositionHistoriesController < ApplicationController
+  def new
+    @employee = Employee.find(params[:employee_id])
```

Comment:
`@employee = Employee.find(params[:employee_id])` is executed for both endpoints at the beginning. [DRY](https://uk.wikipedia.org/wiki/Don%27t_repeat_yourself)